orte/mca/rml/base/rml_base_stubs.c

```diff
@@ -77,16 +77,20 @@ void orte_rml_API_finalize(void)
 {
     orte_rml_base_active_t *active;
 
-    OPAL_OUTPUT_VERBOSE((1,orte_rml_base_framework.framework_output,
+    opal_output_verbose(10,orte_rml_base_framework.framework_output,
                          "%s rml:base:finalize()",
-                         ORTE_NAME_PRINT(ORTE_PROC_MY_NAME)));
+                         ORTE_NAME_PRINT(ORTE_PROC_MY_NAME));
 
     /* cycle thru the actives and see who can send it */
     OPAL_LIST_FOREACH(active, &orte_rml_base.actives, orte_rml_base_active_t) {
         if (NULL != active->module->finalize) {
             active->module->finalize();
         }
     }
+    //[A] 
+    //3/25 -> moving here to avoid multiple calls from each plugin to this base fn
```

Comment:
Just curious: what's "3/25" in reference to?  If this is to be a long-lived comment, something a bit more descriptive would be helpful.